x-pack/plugins/security_solution/public/common/components/event_details/table/add_to_timeline_cell.tsx

```diff
@@ -30,14 +36,48 @@ const AddToTimelineCell = React.memo<AlertSummaryRow['description']>(
       values,
     });
 
+    const clearTimeline = useCreateTimeline({
+      timelineId: TimelineId.active,
+      timelineType: TimelineType.default,
+    });
+
+    const configureAndOpenTimeline = React.useCallback(() => {
+      if (actionCellConfig?.dataProvider) {
+        // Reset the current timeline
+        clearTimeline();
+        // Update the timeline's providers to match the current prevalence field query
+        dispatch(
+          updateProviders({
+            id: TimelineId.active,
+            providers: actionCellConfig.dataProvider,
+          })
+        );
+        // Only show detection alerts
+        // (This is required so the timeline event count matches the prevalence count)
+        dispatch(
+          sourcererActions.setSelectedDataView({
+            id: SourcererScopeName.timeline,
+            selectedDataViewId: 'security-solution-default',
+            selectedPatterns: ['.alerts-security.alerts-default'],
+          })
+        );
+        // Lock the time range by default
+        dispatch(inputsActions.removeGlobalLinkTo());
```

Comment:
Should the lock icon in the timeline display as locked? I think even though the dates matched, it didn't show they were linked

Reply:
Yea that's fine so the user can independently change that range without affecting the global one

Reply:
Updated: https://github.com/elastic/kibana/pull/131255/commits/f758547524289d6b44dc4efd58f92f232b9d35ce

Reply:
I was just testing it and it looks like it's locked even though the lock icon shows they're not linked

Reply:
Updating the timerange in flyout updates it in the global timerange, but not vice versa

Reply:
I looked into it and this appears to be a general bug that was not introduced by these changes. I was able to reproduce the same behaviour by starting a new timeline from the alerts table. I would prefer fixing that issue in a separate PR based on a new ticket, wdyt @michaelolo24 ?

Reply:
fyi @kqualters-elastic this might be worth investigating in https://github.com/elastic/kibana/issues/124713

Reply:
When the new `Investigate in timeline` action for the `Alert prevalence` column is invoked, the status of the lock icon is _unlocked_, which is what I would expect as a user. (This is consistent with the behavior of the row-level  `Investigate in timeline` action in the alerts table.)